.github/workflows/trivy.yml

```diff
@@ -0,0 +1,26 @@
+name: Trivy
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+permissions: read-all
+
+jobs:
+  trivy:
+    name: Run Trivy for IaC Security Analysis
+    runs-on: ubuntu-20.04
```

Comment:
Can we use 22.04 or 24.04?
* https://github.com/actions/runner-images
* https://ubuntu.com/about/release-cycle#ubuntu

Reply:
24.04 in runners is on GitHub's roadmap but not ready yet: https://github.com/github/roadmap/issues/958

Reply:
Using 22.04.